africanlii/apps.py

```diff
@@ -1,4 +1,5 @@
 from django.apps import AppConfig
+from django.utils.translation import gettext as _
```

Comment:
I think you need `gettext_lazy` https://docs.djangoproject.com/en/4.1/topics/i18n/translation/#lazy-translations so that the translation isn't applied when the `_` function is called, because we don't know what language to use yet.